cli/bin/grain.js

```diff
@@ -35,11 +35,27 @@ function graincVersion() {
   return exec.grainc("--version", program).toString().trim();
 }
 
-class GraincOption extends program.Option {
-  grainc = true;
+function wrapAction(action, logError = false) {
+  return (...args) => {
+    try {
+      return action(...args);
+    } catch (e) {
+      if (logError) console.error(e);
+      if (program.opts().graceful) {
```

Comment:
Seems like program is undefined here.

Reply:
It's defined above, on line 16.

Reply:
🤦 I was having flashbacks to when we introduced this bug in v0.3